trace/trace.go

```diff
@@ -45,6 +46,36 @@ func writeFields(b *bytes.Buffer, l []Field) {
 	}
 }
 
+func logTraceArgs(traceNum int32, trace *Trace, totalTime time.Duration) []interface{} {
+	result := make([]interface{}, 0, 2*len(trace.fields)+8)
+	result = append(result, "traceNum", traceNum, "traceName", trace.name)
+	for _, field := range trace.fields {
+		result = append(result, field.Key, field.Value)
+	}
+	result = append(result, "startTime", trace.startTime.Format("02-Jan-2006 15:04:00.000"), "totalTime", durationToMilliseconds(totalTime))
```

Comment:
```suggestion
	result = append(result, "startTime", trace.startTime, "totalTime", totalTime)
```
Let's not do serialization  ourselfs, but delegate it to Structured Logging implementation